docker-compose.override.yml

```diff
@@ -137,3 +137,8 @@ services:
             STREAMR_HTTP_URL: http://10.200.10.1:8081/streamr-core/api/v1
             TOKEN_ADDRESS: "0xbaa81a0179015be47ad439566374f2bae098686f"
             REMOTE_SECRETS: "false"
+    platform:
+        container_name: streamr_dev_platform
+        image: streamr/platform:development
+        ports:
+            - "3333:80"
```

Comment:
depends_on e&e? albeit these depends_on tags might become a nuisance later on when trying to run/dev one service without docker.

Reply:
As you said it may be a slight nuisance later on if you want to start up platform with E&E in IntelliJ.

Reply:
Doesnt it just work to stop EE on docker and start it on EE. Most people will be using docker EE.

Reply:
It does work, but is an extra step, and as such, a nuisance in the long-term.